generate/generate_reference_files.py

```diff
@@ -993,6 +1081,269 @@ def generate_siaf_pre_flight_reference_files_niriss(distortion_file_name, verbos
                                      delimiter_pad=' ', bookend=False)
 
 
+# FGS reference files
+def generate_siaf_pre_flight_reference_files_fgs(verbose=False, mode='siaf'):
+
```

Comment:
Another missing docstring here

Reply:
thanks, we can think about implementing pep8 speaks.

Reply:
added